### PR TITLE
CU-864dmfjy9 - Upgrade Ghost Market API to v2

### PR DESCRIPTION
**Pull request by raulduartep**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-864dmfjy9 Upgrade Ghost Market API to v2](https://app.clickup.com/t/864dmfjy9)

File: app/actions/nftGalleryActions.js [hunk 1]
```diff
@@ -39,39 +31,44 @@ const DEFAULT_NFT_GALLERY_RESULTS = { results: [], page: 0, count: 0 }
...
+        tokenId,
+        contract: contract.hash,
+        collection: {
+          name: collection.name,
+        },
+      }),
+    )
+
+    // eslint-disable-next-line no-console
+    console.log(results)
```
Comment (comountainclimber) [inline app/actions/nftGalleryActions.js]: We should probably remove this log

File: app/actions/nftGalleryActions.js [hunk 2]
```diff
@@ -39,39 +31,44 @@ const DEFAULT_NFT_GALLERY_RESULTS = { results: [], page: 0, count: 0 }
...
-          contract: asset.nft.contract,
-          collection: asset.nft.collection,
-        }
-        return parsed
-      })
+    const { assets, total: count } = response?.data
 
-      return { results: previousResults.concat(results), page, count }
-    }
+    if (!assets || !assets.length || !count) throw new Error('No assets found')
```
Comment (comountainclimber) [inline app/actions/nftGalleryActions.js]: I think keeping the previous pattern of returning an empty payload makes more sense than returning an error here as there is no error that occurred - no NFT's is not really an error

File: app/actions/nftGalleryActions.js (hunk 1 above)
Comment (raulduartep) [reply app/actions/nftGalleryActions.js]: I forgot to remove this log. Solved

File: app/actions/nftGalleryActions.js (hunk 2 above)
Comment (raulduartep) [reply app/actions/nftGalleryActions.js]: Ok. Solved

**Review by comountainclimber — APPROVED** [review]

**Pull request merged**